project/R/conceptSets.R

```diff
@@ -38,46 +272,175 @@ conceptSets <- list(
-      4293619, # source: Tricuspid valve repair NEC /// concept: Repair of tricuspid valve
-      4017751, # source: Unspecified correction of total anomalous pulmonary venous connection /// concept: Repair of total anomalous pulmonary venous connection
-      4019929, # source: Repair of tetralogy of fallot using transannular patch /// concept: Repair of tetralogy of Fallot
-      4020508, # source: Correction of persistent sinus venosus /// concept: Repair of sinus venosus
-      4018926, # source: Other specified : repair of pulmonary artery /// concept: Repair of pulmonary artery
-      4018441, # source: Correction of partial anomalous pulmonary venous drainage /// concept: Repair of partial anomalous pulmonary venous connection
-      4187380, # source: Primary repair of defect of atrioventricular septum NEC/repair of persistent ostium primum/repair of persistent ostium primum /// concept: Repair of ostium primum defect
-      4312194, # source: Aortic valve repair NEC /// concept: Repair of heart valve
-      4049979, # source: Repair of double outlet right ventricle /// concept: Repair of double outlet right ventricle
-      4021725,  # source: Removal of band from pulmonary artery /// concept: Removal of band from pulmonary artery
-      44790092, # source: Relief of left ventricular outflow tract obstruction /// concept: Relief of left ventricular outflow tract obstruction
-      4019028, # source: Release of vascular ring of aorta /// concept: Release of vascular ring of aorta
-      40486525, # source: Primary repair of defect of atrioventricular septum NEC/repair of persistent ostium primum/repair of persistent ostium primum /// concept: Primary repair of defect of atrioventricular septum
-      4217615, # source: Plication of diaphragm /// concept: Plication of diaphragm
-      4020812, # source: Plastic repair of aorta and end to end anastomosis of aorta /// concept: Plastic repair of aorta and end-to-end anastomosis of aorta
-      4019026, # source: Other specified: plastic repair of aorta /// concept: Plastic repair of aorta
-      44510968  # source: Other specified other operations on ventricles of heart /// concept: Other specified other operations on ventricles of heart
+    # 5.22 vital signs UC2
+    vitalSignsUC2 = c(
+      4096101, # source: SpO2 /// concept: Measurement of oxygen saturation at periphery
+      3025315, # source: Weight /// concept: Body weight
+      3020891, # source: Body temperature /// concept: Body temperature
+      3012888, # source: DBP (Diastolic blood pressure) /// concept: Diastolic blood pressure
+      3004249, # source: SBP (Systolic Blood Pressure) /// concept: Systolic blood pressure
+      3014315, # source: Urine output /// concept: Urine output
+      3024171, # source: RR /// concept: Respiratory rate
+      1004025 # source: HR /// concept: Heart rate
```

Comment:
This is a LOINC classification concept. Use [Heart rate](https://athena.ohdsi.org/search-terms/terms/3027018)